knotx-performance-tests/config-knotx/author-config.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
We did not used and tested author but I moved whole configuration to have it documented.